rust-runtime/aws-smithy-http/src/endpoint.rs

```diff
@@ -100,3 +82,23 @@ fn merge_paths<'a>(endpoint: &'a Uri, uri: &'a Uri) -> Cow<'a, str> {
         Cow::Owned(format!("{}/{}", ep_no_slash, uri_path_no_slash))
     }
 }
+
+/// Errors related to endpoint resolution and validation
+pub mod error {
+    /// Endpoint resolution failed
+    #[deprecated(
+        since = "0.60.1",
+        note = "Use aws_smithy_runtime_api::client::endpoint::error::ResolveEndpointError instead."
+    )]
+    pub type ResolveEndpointError =
+        aws_smithy_runtime_api::client::endpoint::error::ResolveEndpointError;
+
+    /// An error that occurs when an endpoint is found to be invalid. This usually occurs due to an
+    /// incomplete URI.
+    #[deprecated(
+        since = "0.60.1",
+        note = "Use aws_smithy_runtime_api::client::endpoint::error::InvalidEndpointError instead."
+    )]
+    pub type InvalidEndpointError =
+        aws_smithy_runtime_api::client::endpoint::error::InvalidEndpointError;
+}
```

Comment:
`ResolveEndpointError` is used by generated code (and I don't see this CR updating that. I'm surprised tests didn't fail from the deprecation) (which makes me concerned)

Reply:
I don't think it's used by generated code, but it is used by the orchestrator.